src/Job.ts

```diff
@@ -43,21 +43,16 @@ export class Job<DATA = any | void> {
 		args: Partial<IJobParameters<DATA>> & {
 			name: string;
 			type: 'normal' | 'single';
-			data: any;
+			data: { [key: string]: any };
```

Comment:
unsure, key could also be a number or a symbol.. I guess. What's the advantage here? The type could be set by generic construct DATA, data as any is internal used only. Or do I miss something?

Reply:
i will revert it as there could be implications